### PR TITLE
committing updated docs

### PR DESCRIPTION
**Pull request by William-Yin123**

### CODE REVIEW

**Review by WilliamHYZhang — COMMENTED** [review]
Awesome to see this many new tests! I don't believe the starting and ending backticks nor ```python``` is needed for any of the doctests. Also, ```Examples:``` seems a bit redundant. Otherwise, if all tests pass this should be a great contribution to tensorflow.

**Comment by mihaimaruseac** [comment]
Please follow the guidelines in https://www.tensorflow.org/community/contribute/docs_ref

**Review by mihaimaruseac — APPROVED** [review]

File: tensorflow/python/ops/ragged/ragged_string_ops.py [hunk 1]
```diff
@@ -679,6 +679,14 @@ def ngrams(data,
   be prevented by setting `preserve_short_sequences`, which will cause the op
   to always generate at least one ngram per non-empty sequence.
 
+  Examples:
+
+  >>> tf.strings.ngrams(["TensorFlow", "and", "keras"], 2)
+  <tf.Tensor: id=285, shape=(2,), dtype=string, numpy=array([b'TensorFlow and', b'and keras'], dtype=object)>
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/ragged/ragged_string_ops.py]:
```
tensorflow/python/ops/ragged/ragged_string_ops.py:685: [C0301(line-too-long), ] Line too long (109/80)
```

File: tensorflow/python/ops/ragged/ragged_string_ops.py [hunk 2]
```diff
@@ -679,6 +679,14 @@ def ngrams(data,
...
+  >>> tf.strings.ngrams(["TensorFlow", "is", "a", "python", "library"], 3)
+  <tf.Tensor: id=320, shape=(3,), dtype=string, numpy=
+  array([b'TensorFlow is a', b'is a python', b'a python library'], dtype=object)>
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/ragged/ragged_string_ops.py]:
```
tensorflow/python/ops/ragged/ragged_string_ops.py:688: [C0301(line-too-long), ] Line too long (81/80)
```

File: tensorflow/python/ops/string_ops.py [hunk 3]
```diff
@@ -78,6 +78,11 @@ def regex_full_match(input, pattern, name=None):
 def regex_replace(input, pattern, rewrite, replace_global=True, name=None):
   r"""Replace elements of `input` matching regex `pattern` with `rewrite`.
 
+  Examples:
+  >>> tf.strings.regex_replace(["python library", "python", "Python language"], "python", "TensorFlow")
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/string_ops.py]:
```
tensorflow/python/ops/string_ops.py:82: [C0301(line-too-long), ] Line too long (103/80)
```

File: tensorflow/python/ops/string_ops.py [hunk 4]
```diff
@@ -78,6 +78,11 @@ def regex_full_match(input, pattern, name=None):
...
+  <tf.Tensor: id=328, shape=(3,), dtype=string, numpy=
+  array([b'TensorFlow library', b'TensorFlow', b'Python language'], dtype=object)>
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/string_ops.py]:
```
tensorflow/python/ops/string_ops.py:84: [C0301(line-too-long), ] Line too long (82/80)
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Please run `pylint`, there are 4 lines that are too long

**Comment by mihaimaruseac** [comment]
Also, I think the `id` of the Tensor should be removed:

```
Expected:
    <tf.Tensor: id=345, shape=(), dtype=float32, numpy=1.55>
Got:
    <tf.Tensor: shape=(), dtype=float32, numpy=1.55>
```

File: tensorflow/python/ops/ragged/ragged_string_ops.py (hunk 1 above)
Comment (William-Yin123) [reply tensorflow/python/ops/ragged/ragged_string_ops.py]: Fixed it.

File: tensorflow/python/ops/ragged/ragged_string_ops.py (hunk 2 above)
Comment (William-Yin123) [reply tensorflow/python/ops/ragged/ragged_string_ops.py]: Fixed it.

File: tensorflow/python/ops/string_ops.py (hunk 3 above)
Comment (William-Yin123) [reply tensorflow/python/ops/string_ops.py]: Fixed it.

**Review by mihaimaruseac — DISMISSED** [review]

File: tensorflow/python/ops/string_ops.py (hunk 4 above)
Comment (William-Yin123) [reply tensorflow/python/ops/string_ops.py]: Fixed it.

**Comment by William-Yin123** [comment]
I noticed a file caused the check to fail so I fixed it. Please approve again if needed.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by William-Yin123** [comment]
I fixed the other failed doc tests that I did not see last time. Please approve again.

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by gbaned** [comment]
@William-Yin123 Can you please resolve conflicts? Thanks!

**Comment by William-Yin123** [comment]
I resolved the conflicts. Can you approve again.

**Review by mihaimaruseac — DISMISSED** [review]
It seems that by resolving conflicts you meant removing the changes you made to `StringUpper`, `StringLower` and `StringToNumber`.

But approving the other changes as they seem ok

**Comment by gbaned** [comment]
@William-Yin123  Can you please resolve conflicts? Thanks!

**Review by mihaimaruseac — APPROVED** [review]

**Comment by William-Yin123** [comment]
I do not know what is causing these errors. I would appreciate it if someone could help me.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
Can you rebase on master please?

**Comment by gbaned** [comment]
@William-Yin123 Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Review by mihaimaruseac — APPROVED** [review]

File: tensorflow/core/api_def/base_api/api_def_AsString.pbtxt [hunk 5]
```diff
@@ -42,5 +42,11 @@ Supports many numeric types and boolean.
 For Unicode, see the
 [https://www.tensorflow.org/tutorials/representation/unicode](Working with Unicode text)
 tutorial.
+
+Examples:
+>>> tf.strings.as_string([3, 2])
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_AsString.pbtxt]: There needs to be a newline between `Examples:` and the doctest. Please do so in all files.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/core/api_def/base_api/api_def_AsString.pbtxt (hunk 5 above)
Comment (mihaimaruseac) [reply tensorflow/core/api_def/base_api/api_def_AsString.pbtxt]: Fixed in `dad8c5d`

**Pull request merged**